index.html

```diff
@@ -1,252 +1,248 @@
+                <div class="mycanopy-section">
+                    <div class="snapshoot-portofolio">
+                        <img src="img/Snapshoot Portfolio(13).png" alt="snapshoot Portfolio picture">
+                    </div>
+                    <div class="project-card">
+                        <h2>Facebook 360</h2>
+                        <h3>FACEBOOK &nbsp; <img src="img/Counter.png" alt="dot symbol">&nbsp;<span> Full Stack Dev
+                                &nbsp; <img src="img/Counter.png" alt="dot symbol"> &nbsp; 2015 </span></h3>
+                        <p>Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.</p>
+                        <ul>
+                            <li class="html-tag">html</li>
+                            <li class="ruby-tag">Ruby on rails</li>
+                            <li class="css-tag">css</li>
+                            <li class="js-tag">javascript</li>
+                        </ul>
+                        <button type="button">See Project</button>
+                    </div>
                 </div>
-                <div class="project-card">
-                    <h2>Multi-Post<br />Stories</h2>
-                    <h3>CANOPY &nbsp;<img src="img/Counter.png" alt="dot symbol"> &nbsp;<span> Back End Dev &nbsp;<img src="img/Counter.png"alt="dot symbol"> &nbsp; 2015</span></h3>
-                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-                    <ul>
-                        <li>html</li>
-                        <li>css</li>
-                        <li>javascript</li>
-                        <button type="button">See Project </button>
-                    </ul>
+            </div>
+
+            <div class="canopy-card-section">
+                <div class="mycanopy-section">
+                    <div class="project-card">
+                        <h2>Uber Navigation</h2>
+                        <h3>Uber &nbsp; <img src="img/Counter.png" alt="dot symbol">&nbsp;<span> Lead Developer
+                                &nbsp; <img src="img/Counter.png" alt="dot symbol"> &nbsp; 2018 </span></h3>
+                        <p>A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.</p>
+                        <ul>
+                            <li class="html-tag">html</li>
+                            <li class="ruby-tag">Ruby on rails</li>
+                            <li class="css-tag">css</li>
+                            <li class="js-tag">javascript</li>
+                        </ul>
+                        <button type="button">See Project</button>
+                    </div>
+                    <div class="snapshoot-portofolio">
+                        <img src="img/Snapshoot Portfolio(14).png" alt="snapshoot Portfolio picture">
+                    </div>
```

Comment:
- You did a great job in your design but kindly make sure you use the appropriate path to display your image on my side the image is broken please fix that  :+1: 

## Your design 
![image](https://user-images.githubusercontent.com/95297251/221247008-ffc00842-d987-4823-a6c3-39246325bf4b.png)